.freeCodeCamp/tooling/test-utils.js

```diff
@@ -243,6 +244,10 @@ async function parseJs(jsString) {
   return acornParse(jsString, { ecmaVersion: 2020, sourceType: 'module' });
 }
 
+async function babeliser(codeString) {
+  return new Babeliser(codeString);
+}
+
```

Comment:
Reminder: Editing this file does nothing for releases - **DO NOT** edit anything `.freeCodeCamp` if you need it in production